src/elements/common/annotator-context/withAnnotatorContext.tsx

```diff
@@ -9,7 +9,9 @@ export interface WithAnnotatorContextProps {
 export default function withAnnotatorContext<P extends {}>(WrappedComponent: React.ComponentType<P>) {
     return React.forwardRef<React.RefForwardingComponent<React.ComponentType<P>>, P>((props, ref) => (
         <AnnotatorContext.Consumer>
-            {annotatorState => <WrappedComponent ref={ref} {...props} annotatorState={annotatorState} />}
+            {({ activeAnnotationId, ...rest }) => (
```

Comment:
Why is this particular property separated out? Should we just spread `{...annotatorState}`, instead?